srcpkgs/glibc/INSTALL

```diff
@@ -0,0 +1,10 @@
+# Build iconv module cache
+#
+case "${ACTION}" in
+post)
+	if [ -e usr/lib64 ]; then
+		echo "Generating iconv module cache..."
+		iconvconfig --nostdlib --prefix="$(pwd)" usr/lib64/gconv
```

Comment:
For the normal case, don't run this conditionally, and always use `usr/lib`.

What you can do to cater to (current) multilib setups is detect if `usr/lib32` is a directory and `usr/lib32/gconv` exists, and then run iconvconfig for it as well, assuming it behaves correctly (which it should, glibc is full of workarounds for every thing).